src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(String fruit, int quantity) {
+        Storage.storage.put(fruit, quantity);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return Storage.storage.get(fruit);
+    }
+}
```

Comment:
According to the encapsulation principle, consider hiding the direct access to the `Storage.storage` map. Instead, provide methods in the `Storage` class to interact with the storage data. This will encapsulate the logic of data storage and manipulation within the `Storage` class and will make it easier to handle any changes in the storage implementation.